notebooks/HASP/COSLifetimePositions/COSLifetimePositions.ipynb

```diff
@@ -0,0 +1,1844 @@
irMWDy+HF7VRwe8Dt0+JR9Hg1JhTNgU+h1PrdGLwODD4bBsUeuKk2tBo7Wr0dVW/DZ3TgNjlwBDtpsdhpCHFiNbmx6sCm1WDX7AzG3AcpfFIVwx7DMNVvQdnxOEgbTqghgghTBBFBoUQGGwkPMhAeZCQi2EB4kIGIoMB9eLCBiB3PW4w6eRMUQgghhBDHFb+itgZrVpcX246wrcXl3XHsr8d9WN1umt0tWD2N2P1NOJUmFI0VjT4QvGl3BHCaP8I4re/vO7Ifwj1G4lyRRLmiiXBFEuKOJtgTjdEdjd4bg0b5+8DtD3q/E+MfIZu7GZO7uTV4+3MQp/f/zQg+vR5dSAjakJD/b+/eY+Sq6/+PP8/9nJndnbZUehG6IgoCVbSt4SIXJaZcAlQhWhNsIMFGYrgVaAwI0RATJAooUQEjfCORCDGEiwGrmHANlcQCNSgBJWiBtPRHpd2dy7l/fn/M7HSX7ra73W3Ztq9HcnJmPnPm9DPNu+92X/3MOdjV6vatp713qlXs6rDXenqwq5X2az0jx60g0M8lB6A9nRNN7gJLInuZKUvKZpOyXqes18kHBmn9r068tbNKbCAhruckzZw4KUkSQ5rZpIVLalxSKyCzIwonGONXcBn1j8WwBWZWWeDlDdysjls2sMsGNg2M1aC0G2Ruo31tMb/BYNhga9TkvWqLzT0570UWqTeR1WoWsPOvfZrSxxQhpgyhDLqPTRlAdzwksCuETpWKW6XH62NmOJODwpnMqvYwI/KpRW43CKtVtgdkfZGnO16KiIiIiIyTY1ud/3z2gPEHUcMZY0jycnv4NiyI29bKqCcx29JBBpM6g2mdelanmTVo5g1aRYOkaBKXTbKySWaaFMQUtDBWC9tuYdkJxknAKgHY5qds89+FvndHmQyEeZVKWqOS9VFNa1TSPipZH5W0j2pnX0lruMYjdyLyKKIZzdnFh0ywi224+QB+NkCYDBC1tlFpbiWK20Gc1xjE2/oOFpNcB+Q43TDOqVaxK9Uxg7odArlhoZ3TU9VXWKVLgZrsMSbL2uFXq0XZbFG2mphWqz3WbLXHO2NFo0XaiIkbOUlctFeHJYY4pROIOZ1VYmHnbpTVzt0oo87dKP3ONgqbEYHY9gmWuHkTN29gFw0s08SYBrndILWbxG6Dlt+k6bWDsYGoyfvVJtuihMEKZN5Em+jI6xaY0m2HXqXfDsCKTiDWeWzKEIeoE4RVqLg9VNwe+oIeevweakEvM8JeamH74vs9oUdP4NIbup3nLr2dfeRppZiIiIiIyL7CsixCzyH0HD7SO9ZigIkzxpAWJXFa0sxyBlottsQDvN8aYGs8yLa4zkA62A7p0jr1rEErb9DM66RlQl402VoO8r9yM6WJKUkpSCnJyE2OUwb4WQ+VvBO6pX1Us+2Ph0I4vwzBCijdg0ndg0lDqPeOPe/calJYDUoaGNPAKts/w7l5Ay9r4GcNgqRBFNepxA2qrQbVVkpl6GatRUE5MEA5MMBk1/YZy6IMQ0ylglVph3NOTy9OTxW3t4rf24vbM76Vc3alguXo+tD7Kn3l8wBljIEso4xjylaMiVuUcdwOvOKEMm5hhr82LAAbCsSKZoO82aBoNjrjrXY4FudkhUdhBWRelczrIXMrncdV8k4YlnnV7njuVjH27jcSq2xB2QDTpKBBYTfI7Cap02gHY16Dpt+kHjQZDJsMRA22Vlqknhlxi+odzmvAK20c42CXLlbpYUqfsgwoTEhWVEhNBVMGmNLDMgG+HRI4EaEbETkRFa9CxavQ41fo9Sv0+VV6gwp9YUAlcNrhVzA8BPPoCV2qgaOL74uIiIiIyD7FGENapsR5TJzHDKZNBpMWg6069db/aDW20BzYRuv9Jmk9I6sXlC2HMnYhCbCTCnbWg5P14BSV3Z5HbmUkboPUaZLZ7YUT3UDONLCKBm7RwMkb+GmDIG0QJk0qaUmUQJgaohSihPY+BXsPpCep55IHPnkYUEQhJqpgVSrY1R68nh783hpRrUbUVyOs9eEOX1VXrW4P66paPfdBuobaOO0vgdoOQVcyLNRqxSODrlFeK1sxRasddOVxJwDrvi+BOMFKUqwkwyrLsecBFE5A4YRkboW8uyqsQu5VuyvEss7j3Kt0H5fOGCvFxvX5U0qrQW41ye0GidMgcRu0vBYNv0EjaFAPGmR+HeM2wGlg23UCCgJjCI3BNRZ26WAZF0qX0nidACwgK0NSE5KUFVJTpbB7Ke0ahVvD9mpU3ApVP6LHr9LnV+kJQnoCj2rQDriqfntf8d0PjLlUfIfAtdXAREREREREpkBRlCSNnLieEW9rEG/dRrxtkHigSTIYE9dT4mZOq1nSii3ixCXJfIzZ/YUJpd2icGIKOyZ32lvqJKR2QmbF5FZCYRKMibHLGKdIsIsYP4vxsgQ/axEmMVGaEKVmZDg3LKBzx/5xfPfnDqS+Sxq4ZKFPHviUUYiJIqhEONUqTrUHv6+PsLdGdcYMqjNmEvbNwO+pfeCrrtV9/tpzuobaNGK6QdcHVm/FyeiBV2esHXy1KFotsmadotWkaHVWfCUxdIIukgQr3nnQNV6F7VI4Ebk7tPWSuweTRxF5T0Tuht3XMjci89r73I0onJDSicDa/SZkKMidBoVbp3Ca4NbBqWPZdRynjmMP4tl1fKeObw0S2XVCa4CKlRIag19CYXySMiI3FVKrh5w+cruXzNTIzAJS+sidGRThDAhnQjQDqzKLKIqIvHboVfGdzuYS+U47DPPaj31X1wUTERERERGZrhzHptLnU+nzYX4VOHiX7zHGkCUFcSPbHsbVE+Jt9fY20Go/b2TEzZK4BUlik2TteMQuI+wywtvdSXudDYitmMSOwW5h7BjjxBROSm6nZHZCQUZBSmFSTJliigxTpNhFip2nuHmClyUEWUaYpgRpRpRkRHHaXkmXQphBmGy/0lGY5oRpDoPxLqeaAVt38nphQeLbJEEnpAs88tCnCANMFEIlwqpWcHp
```

Comment:
same here, replace with tbdata=fits.getdata so you can remove hdulist and hdulist.close()